lib/sda/host_msg.go

```diff
@@ -61,10 +67,13 @@ type TokenID uuid.UUID
 type Token struct {
 	EntityListID EntityListID
 	TreeID       TreeID
-	ProtoID      ProtocolID
-	RoundID      RoundID
-	TreeNodeID   TreeNodeID
-	cacheId      TokenID
+	// TO BE REMOVED
```

Comment:
So `ProtoID`, `ServiceID` and `RoundID` will all be stored in `TreeNode`?

BTW: why not `ProtocolID`?